### PR TITLE
Hw 43 palindrom

### PR DESCRIPTION
**Pull request by TarasHryts**

### CODE REVIEW

File: src/main/java/core/basesyntax/Palindrome.java [hunk 1]
```diff
@@ -21,6 +21,13 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String letterNumber = text.replaceAll("\\W", "").toLowerCase();
+        boolean flagCheck = true;
+        for (int i = 0; i < letterNumber.length() / 2; i++) {
+            if (letterNumber.charAt(i) != letterNumber.charAt(letterNumber.length() - 1 - i)) {
+                flagCheck = false;
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Palindrome.java]: you can return false immediately

Comment (TarasHryts) [reply src/main/java/core/basesyntax/Palindrome.java]: done

File: src/main/java/core/basesyntax/Palindrome.java [hunk 2]
```diff
@@ -21,6 +21,12 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String letterNumber = text.replaceAll("\\W", "").toLowerCase();
+        boolean flagCheck = true;
+        for (int i = 0; i < letterNumber.length() / 2; i++) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Palindrome.java]: we can use method reverse() to simplify this code

Comment (TarasHryts) [reply src/main/java/core/basesyntax/Palindrome.java]: done

**Pull request closed without merging**